src/app/(registration)/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -2,9 +2,28 @@
 "use client";
 
 import { SignIn } from "@clerk/nextjs";
+import { OakBox } from "@oaknational/oak-components";
 
 import { formAppearanceStyles } from "../../formAppearanceStyles";
+import { AuthLayout } from "../../auth-layout";
+
+import { getIllustrationAsset } from "@/image-data";
+import CMSImage from "@/components/SharedComponents/CMSImage";
 
 export default function SignInPage() {
-  return <SignIn appearance={formAppearanceStyles} />;
+  return (
+    <AuthLayout
+      asideSlot={
+        <OakBox $maxWidth="all-spacing-21">
+          <CMSImage
+            image={getIllustrationAsset("auth-acorn")}
+            $width="100%"
+            $objectFit="contain"
+          />
+        </OakBox>
```

Comment:
This is the original "acorn" decoration copied from the original layout. So the sign-in page will be unchanged.